From d17f9f527f2cdae7564602525b7e851ce4902115 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 14 Apr 2018 20:10:12 +0200
Subject: [PATCH 16/23] format-macros.h: do not rely on __STDC_FORMAT_MACROS

The PRId64 macro and friends are not universally defined in inttypes.h.
Let's just go ahead and define them when needed.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 deps/v8/src/base/format-macros.h | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/deps/v8/src/base/format-macros.h b/deps/v8/src/base/format-macros.h
index 5f5fe5df..13f240a2 100644
--- a/deps/v8/src/base/format-macros.h
+++ b/deps/v8/src/base/format-macros.h
@@ -80,6 +80,16 @@
 
 #else  // V8_OS_WIN
 
+#ifndef PRId64
+#define PRId64 "I64d"
+#endif
+#ifndef PRIu64
+#define PRIu64 "I64u"
+#endif
+#ifndef PRIx64
+#define PRIx64 "I64x"
+#endif
+
 #if !defined(PRId64) || !defined(PRIu64) || !defined(PRIx64)
 #error "inttypes.h provided by win toolchain should define these."
 #endif
-- 
2.17.0.windows.1

